deps/v8/src/api.cc

```diff
@@ -939,6 +939,15 @@ void Template::Set(v8::Local<Name> name, v8::Local<Data> value,
   i::Isolate* isolate = templ->GetIsolate();
   ENTER_V8(isolate);
   i::HandleScope scope(isolate);
+  auto value_obj = Utils::OpenHandle(*value);
```

Comment:
Oh, it's not going to throw; newer V8 versions replace it with a CHECK that aborts when it fails.